### DISCUSSION TITLE
Netflix QC - dual speaker is checked with a space. Guideline suggests without space

### DISCUSSION THREAD

**Discussion by PreetM07** [SubtitleEdit/subtitleedit#5147] [Q&A] [answered]
hello,

Netflix QC check in SE considers dual speaker with a space, however the template guideline suggests without space.
Do you think there is something that I am missing?
https://partnerhelp.netflixstudios.com/hc/en-us/articles/219375728-Timed-Text-Style-Guide-Subtitle-Templates

**Comment by OmrSi** [comment, marked as answer]
Hey, @PreetM07 
Is that happening for a specific language?
Because it's working well for English here:

![image](https://user-images.githubusercontent.com/20923700/122914930-db55fa00-d363-11eb-8eee-e9134534731b.png)

**Comment by PreetM07** [comment]
Hi @OmrSi 

I see. Yes, it was for Czech. Which is correct there. Thanks.
